From b69a3d5dd9499642b2c0aa36aacf4e516c2b8dd0 Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Tue, 15 Dec 2009 10:09:07 -0600
Subject: [PATCH 140/449] ENGR00119264 [MX25] SCC security unit test fails

Added checks to Makefile to only copy scripts from SCC and SCC2 folder if platform is not in the excluded list

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 test/mxc_scc/Makefile  | 8 +++++---
 test/mxc_scc2/Makefile | 7 ++++---
 2 files changed, 9 insertions(+), 6 deletions(-)

diff --git a/test/mxc_scc/Makefile b/test/mxc_scc/Makefile
index ec74db9..92c0ff5 100644
--- a/test/mxc_scc/Makefile
+++ b/test/mxc_scc/Makefile
@@ -1,14 +1,16 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS MXC30031ADS"
+EXCLUDE_LIST:="IMX21ADS MXC30031ADS IMX233 IMX37_3STACK IMX51"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
-OBJS = $(OBJDIR)/scc_test $(OBJDIR)/autorun-scc.sh
+OBJS = $(OBJDIR)/scc_test
 else
 endif
 
 
 all : $(OBJS)
-	cp -rf *.sh $(OBJDIR)
+ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
+	cp -r *.sh $(OBJDIR)
+endif
 
 .PHONY: clean
 clean :
diff --git a/test/mxc_scc2/Makefile b/test/mxc_scc2/Makefile
index 85e8f04..cb418ec 100644
--- a/test/mxc_scc2/Makefile
+++ b/test/mxc_scc2/Makefile
@@ -1,16 +1,17 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK IMX233"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
-OBJS = $(OBJDIR)/scc2_test.out $(OBJDIR)/autorun-scc2.sh
+OBJS = $(OBJDIR)/scc2_test.out
 else
-OBJS =
 endif
 
 CFLAGS += -DFSL_PLATFORM_OTHER -DFSL_HAVE_SCC2
 
 all : $(OBJS)
+ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 	cp -r *.sh $(OBJDIR)
+endif
 
 .PHONY: clean
 clean :
-- 
1.8.0

